examples/eks-msk-opensearch/terraform/templates/fluent-bit.yaml

```diff
@@ -0,0 +1,132 @@
+# Template out the image version and service account name to use so that they can be modified from Terraform.
+image:
+  tag: ${image_version}
+
+serviceAccount:
+  name: ${service_account_name}
+
+# Tolerate all taints so that all nodes are logged.
+tolerations:
+  - key: ""
+    operator: "Exists"
+    effect: "NoExecute"
+
+# Set hostNetwork to true and dnsPolicy to ClusterFirstWithHostNet so that fluent bit DaemonSet could call Kubelet locally.
+hostNetwork: true
+dnsPolicy: ClusterFirstWithHostNet
+
+# Node access required when using Kubelet
+rbac:
+  create: true
+  nodeAccess: true
+
+# Pipeline configuration.
+config:
+  # This is a config for the FluentBit application running the pipeline.
+  # We set the flush interval, log level and where our parsers are stored.
+  # We additionally expose the HTTP service so that Terraform can ping it.
+  service: |
+      [SERVICE]
+        Flush                     30
+        Log_Level                 info
+        Daemon                    off
+        Parsers_File              parsers.conf
+        HTTP_Server               On
+        HTTP_Listen               0.0.0.0
+        HTTP_Port                 2020
+        storage.path              /var/fluent-bit/state/flb-storage/
+        storage.sync              normal
+        storage.checksum          off
+        storage.backlog.mem_limit 5M
+        
+      @INCLUDE application-log.conf
+  extraFiles:
+    application-log.conf: |
+        [INPUT]
+            Name                tail
+            Tag                 application.*
+            Exclude_Path        /var/log/containers/cloudwatch-agent*, /var/log/containers/fluent-bit*, /var/log/containers/aws-node*, /var/log/containers/kube-proxy*
+            Path                /var/log/containers/*.log
+            multiline.parser    docker, cri
+            Parser              docker
```

Comment:
with the `multiline.parser    docker, cri` there is no longer any need to also set `Parser              docker`